modules/installation-osp-creating-bootstrap-machine.adoc

```diff
@@ -23,7 +23,7 @@ Create a bootstrap machine and give it the network access it needs to run on {rh
 ====
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/installer/release-4.6/upi/openstack/bootstrap.yaml[]
+include::https://raw.githubusercontent.com/openshift/installer/release-4.7/upi/openstack/bootstrap.yaml[]
```

Comment:
My preference for this and other included files would to be pop them off into their own platform-specific PRs that QE can review.